### PR TITLE
runfix: Add a wrapper to group creation component to fix broken styles

### PR DESCRIPTION
**Pull request by thisisamir98**
fixes broken automation tests

### CODE REVIEW

**Review by atomrc — APPROVED** [review]

File: src/script/components/UserSearchableList.tsx [hunk 1]
```diff
@@ -161,26 +161,32 @@ const UserSearchableList: React.FC<UserListProps> = ({onUpdateSelectedUsers, ...
...
+      <div className="user-list-wrapper">
+        {users.length === 0 ? (
+          <div className="user-list__no-results" data-uie-name="status-all-added">
+            {t('searchListEveryoneParticipates')}
+          </div>
+        ) : (
+          <div className="user-list__no-results" data-uie-name="status-no-matches">
+            {t('searchListNoMatches')}
+          </div>
+        )}
```
Comment (przemvs) [inline src/script/components/UserSearchableList.tsx:174]: I'll change this functionality. Make users.length === 0 as variable, eg
```
const hasUsers = !!users.length
```
Also make variables for data-uie-name and translation key

```
const dataUieName = hasUsers ? "status-no-matches" : "status-all-added";
const translationText = hasUsers ?  'searchListNoMatches' : 'searchListEveryoneParticipates';
```
And for component without ternary:
```
<div className="user-list__no-results" data-uie-name={dataUieName}>
  {t(translationText)}
</div>
```

**Review by przemvs — COMMENTED** [review]
LGTM, we can consult about this functionality ;)

File: src/script/components/UserSearchableList.tsx (hunk 1 above)
Comment (atomrc) [reply src/script/components/UserSearchableList.tsx:174]: Yes, that's is a valuable refactoring, I agree :+1: 

Also we could refactor to have the `<div className="user-list-wrapper">` only once in the component.

Like having something like 

```ts
const content = emptyUserList ? <div classname="noresults">...</div> : <UserList />

return <div className="user-list-wrapper">{content}</div>;
```

Comment (przemvs) [reply src/script/components/UserSearchableList.tsx:174]: @atomrc it's better to have if statement for return component, it's more readable ;)

Comment (thisisamir98) [reply src/script/components/UserSearchableList.tsx:174]: https://github.com/wireapp/wire-webapp/pull/13159

**Pull request merged**